cfme/rest/gen_data.py

```diff
@@ -86,18 +87,13 @@ def dialog():
         choose_type="Text Box",
         default_text_box="default value"
     )
-    service_dialog = ServiceDialog(
-        label=dialog,
-        description="my dialog",
-        submit=True,
-        cancel=True,
-        element_data=element_data,
-        tab_label="tab_{}".format(fauxfactory.gen_alphanumeric()),
-        tab_desc="my tab desc",
-        box_label="box_{}".format(fauxfactory.gen_alphanumeric()),
-        box_desc="my box desc"
-    )
-    service_dialog.create()
+    service_dialog = service_dialogs.create(label=dialog,
+        description="my dialog", submit=True, cancel=True,)
+    tab = service_dialog.dialogtab.create(tab_label='tab_' + fauxfactory.gen_alphanumeric(),
+        tab_desc="my tab desc")
+    box = tab.dialogbox.create(box_label='box_' + fauxfactory.gen_alphanumeric(),
+        box_desc="my box desc")
+    box.dialog_element.create(element_data=[element_data])
```

Comment:
Should the element data really be a list? I though it should be a dict. Ideally create should take all the keywords required for th eelement data, this my be a bit clunky.